pkg/tenant/auth.go

```diff
@@ -54,14 +56,28 @@ func (s *Server) Register(c *gin.Context) {
 
 	// TODO: Handle error status codes returned by Quarterdeck
 	var reply *qd.RegisterReply
-	if reply, err = s.quarterdeck.Register(c.Request.Context(), req); err != nil {
+	if reply, err = s.quarterdeck.Register(ctx, req); err != nil {
 		log.Error().Err(err).Msg("could not register user")
 		c.JSON(http.StatusInternalServerError, api.ErrorResponse("could not complete registration"))
 		return
 	}
 
 	// TODO: Send verification email to the provided email address
-	// TODO: Create tenant and other Tenant-specific resources for the user
+
+	// Create a partial member record for the new user
+	// Note: Tenant ID is not populated because it hasn't been created yet
+	member := &db.Member{
+		OrgID: reply.OrgID,
+		Name:  req.Name,
+		Role:  reply.Role,
+	}
+
+	// Create a default tenant and project for the new user
+	// Note: This method returns an error if the member model is invalid
+	if err = db.CreateUserResources(ctx, member); err != nil {
+		log.Error().Str("user_id", reply.ID.String()).Err(err).Msg("could not create default tenant and project for new user")
+		// TODO: Does this leave the user in a bad state? Can they still use the app?
```

Comment:
Thanks for making note of how this could impact the frontend. I think we should create a story to look into what will happen if the defaults aren't created. Once we know, we'll then be able to create additional stories, if necessary.

Reply:
Sound good, I'll create the story!